### DISCUSSION TITLE
Move Alerts & Reports to the top level menu

### DISCUSSION THREAD

**Discussion by cemremengu** [apache/superset#19377] [Ideas]
It is a very powerful feature and probably one of the first things people look for in most BI software (especially those in corporate companies where email reports are daily routine).

It deserves to be there instead of being tossed under settings 😄

**Comment by kasiazjc** [comment]
Hi @cemremengu, thanks for the feedback! You are talking about the top navigation, right? I think that it all depends on the users and how much/often they are using this feature, what is the normal flow etc. 🤔 I agree though, that it is quite hidden right now and may be not too intuitive and hard to find at first. I am adding it to our backlog to check! 

Using the opportunity I'd like to ask how are you using Alerts&Reports? How often do you create new ones and what are the features that you use the most on this page? Thank you 🙏

**Comment by cemremengu** [comment]
Thanks @kasiazjc ! For corporate clients `Alerts&Reports` are very important. We use it to sent periodic mail with data attachments (csv, xlsx, etc.).

Unfortunately, report mails with attachments are the  only thing they care about in these type of companies and event with the coolest features of a BI, it comes to whether it can email reports periodically.

**Reply by kasiazjc** [reply]
Makes sense, thank you for feedback 🙏🏻
